src/main/containers/SourcePanel.jsx

```diff
@@ -1,18 +1,29 @@
 import React, { useState } from 'react';
 import DataCanvas from './DataCanvas.jsx';
 import RequestBar from '../components/RequestBar.jsx';
-import { smallData } from '../dummyData';
 
 const SourcePanel = (props) => {
   const {
     treeCount, updateTreeCount, data, setData,
   } = props;
+  
+  const dataTreeOptions = {
```

Comment:
can we make these callbacks?  the callback is what allows us to access what was actually mutated from the original. Nothing is wrong with setting these as booleans for now, but in the future we would on to access the values that were changed from the user for something later on.

Reply:
We could but it’s more concise to just pass `false`, and that’s what’s used in the library docs. When we want to enable any of these features, we’ll pass down callbacks in the options object instead.